*** /tmp/,RCSt1a05894	Fri Jun 30 15:47:01 1995
--- overhead/malloc/malloc.ci	Fri Jun 30 15:45:43 1995
***************
*** 453,459 ****
  	int x;
  	/* set min to the largest of min, 1<<14, arena size >>3 */
  	if (min < 1<<14) min = 1<<14;
! 	x = (A.arenaend - A.arenastart)>>3;
  	if (A.allocp != 0 && min < x) min = x;
  	/* round request to a full page  (though not aligned on page boundary )*/
  	segbytes = ((min+2*EPSILON+(SEGGRAIN-1)) 
--- 453,459 ----
  	int x;
  	/* set min to the largest of min, 1<<14, arena size >>3 */
  	if (min < 1<<14) min = 1<<14;
! 	x = ((char *)A.arenaend - (char *)A.arenastart)>>3;
  	if (A.allocp != 0 && min < x) min = x;
  	/* round request to a full page  (though not aligned on page boundary )*/
  	segbytes = ((min+2*EPSILON+(SEGGRAIN-1)) 

